cmd/tempo/app/modules.go

```diff
@@ -422,10 +423,15 @@ func (t *App) initQueryFrontend() (services.Service, error) {
 	// http endpoint to see usage stats data
 	t.Server.HTTPRouter().Handle(addHTTPAPIPrefix(&t.cfg, api.PathUsageStats), usageStatsHandler(t.cfg.UsageReport))
 
+	t.Server.HTTPRouter().PathPrefix("/static/").HandlerFunc(http.FileServer(http.FS(staticFiles)).ServeHTTP).Methods("GET")
```

Comment:
i am struggling to parse this line and I think it may be the most critical in this PR. adding a static file server to Tempo is a security concern, but it looks like this may only be allowed to access preloaded files in memory?

Reply:
Right, the determination is at compile time, and read only.